worldedit-bukkit/src/main/java/com/sk89q/worldedit/bukkit/adapter/BukkitImplAdapter.java

```diff
@@ -112,7 +112,7 @@
      * @param pos The position
      * @param nbtData The NBT Data
      */
-    void sendFakeNBT(Player player, Vector pos, CompoundTag nbtData);
+    void sendFakeNBT(Player player, Vector3 pos, CompoundTag nbtData);
```

Comment:
Should this be BlockVector3? It'd require an adapter change too though.

Reply:
Yea, we need an adapter change anyways to change to `Vector3`. Will update.